app/controllers/miq_policy_controller.rb

```diff
@@ -771,10 +771,12 @@ def handle_selection_buttons(members,
           {:member => members.to_s.split("_").first.titleize}, :error)
       else
         mems = @edit[choices].invert
-        if @edit[:event_id]                                           # Handle Actions for an Event
+        if @edit[:event_id]
```

Comment:
How?  Move `mems = @edit[choices].invert` line inside both branches?  Don't think we want more duplication.